scene/main/node.cpp

```diff
@@ -840,6 +844,8 @@ bool Node::is_processing_internal() const {
 void Node::set_process_priority(int p_priority) {
 	data.process_priority = p_priority;
 
+	ERR_FAIL_COND(!data.tree);
```

Comment:
This now causes an error when loading a project in the editor:
```
ERROR: set_process_priority: Condition ' !data.tree ' is true.
   At: scene/main/node.cpp:847.
```

Reply:
Hmm, what condition are you testing to get this error? If you hadn't taken this error, editor would crash in one of the 4 if structures. According to your [comment](https://github.com/godotengine/godot/pull/29980#discussion_r296474512), is there a condition that must not enter the any of structures?

Reply:
Just open the editor on any project:
```
$ mkdir ~/tmp/new_project
$ cd ~/tmp/new_project
$ touch project.godot
$ godot -e
Godot Engine v3.2.dev.custom_build.266dc732b - https://godotengine.org
Found discrete GPU, setting DRI_PRIME=1 to use it.
Note: Set DRI_PRIME=0 in the environment to disable Godot from using the discrete GPU.
OpenGL ES 3.0 Renderer: AMD VEGAM (DRM 3.30.0, 5.1.14-desktop-1.mga7, LLVM 8.0.0)
 
ERROR: set_process_priority: Condition ' !data.tree ' is true.
   At: scene/main/node.cpp:847.
```

Reply:
I tried your way and also i created new project from editor. And I didn't get that error on version `v3.2.dev.custom_build. 7d8d337b2`.

Reply:
@bojidar-bg fixed it in #30254, so no worry :)